app/components/public/social-links.js

```diff
@@ -9,5 +10,13 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+   calender:computed('eventtime','eventtimeends','eventname','eventlocation',function(){
```

Comment:
Codacy found an issue: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)